### DISCUSSION TITLE
How to run GDevelop Engine

### DISCUSSION THREAD

**Discussion by cauzhjsj** [4ian/GDevelop#2403] [Enhancement ideas (and their technical discussions)]
How to run GDevelop Engine

**Comment by Bouh** [comment]
http://wiki.compilgames.net/doku.php/gdevelop5/getting_started

**Reply by 4ian** [reply]
Everything is indeed on the wiki. If you're a developer and interested in contributing, read the GitHub README that explains everything about how to start.
I'm locking now as discussions are about technical subjects only. Prefer the forum/Discord for anything else.
